common/client2/src/main/java/com/bazaarvoice/emodb/client2/EmoClientException2.java

```diff
@@ -0,0 +1,24 @@
+package com.bazaarvoice.emodb.client2;
```

Comment:
here and in other classes, changing package name to client2 is enough, don't need to change the name of the classes